@dhkatz 
This port currently should not support `arm`.
The errors like this:
```
4>C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.25.28610\include\emmintrin.h(24,1): fatal error C1189: #error:  This header is specific to X86 and X64 targets (compiling source file C:\agent\_work\1\s\buildtrees\v-hacd\src\e17ceb9aef-3a5e115dd0\src\VHACD_Lib\src\btAlignedAllocator.cpp) [C:\agent\_work\1\s\buildtrees\v-hacd\arm-uwp-dbg\src\VHACD_Lib\vhacd.vcxproj]
     4>C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.25.28610\include\emmintrin.h(24,1): fatal error C1189: #error:  This header is specific to X86 and X64 targets (compiling source file C:\agent\_work\1\s\buildtrees\v-hacd\src\e17ceb9aef-3a5e115dd0\src\VHACD_Lib\src\btConvexHullComputer.cpp) [C:\agent\_work\1\s\buildtrees\v-hacd\arm-uwp-dbg\src\VHACD_Lib\vhacd.vcxproj]
     4>C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.25.28610\include\emmintrin.h(24,1): fatal error C1189: #error:  This header is specific to X86 and X64 targets (compiling source file C:\agent\_work\1\s\buildtrees\v-hacd\src\e17ceb9aef-3a5e115dd0\src\VHACD_Lib\src\vhacdMesh.cpp) [C:\agent\_work\1\s\buildtrees\v-hacd\arm-uwp-dbg\src\VHACD_Lib\vhacd.vcxproj]
     4>C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.25.28610\include\emmintrin.h(24,1): fatal error C1189: #error:  This header is specific to X86 and X64 targets (compiling source file C:\agent\_work\1\s\buildtrees\v-hacd\src\e17ceb9aef-3a5e115dd0\src\VHACD_Lib\src\vhacdVolume.cpp) [C:\agent\_work\1\s\buildtrees\v-hacd\arm-uwp-dbg\src\VHACD_Lib\vhacd.vcxproj]
```
So you can add the unsupported info.

--- a/ports/v-hacd/CONTROL
+++ b/ports/v-hacd/CONTROL
@@ -2,9 +2,11 @@ Source: v-hacd
 Version: 3.2.0
 Homepage: https://github.com/kmammou/v-hacd
 Description: The V-HACD library decomposes a 3D surface into a set of "near" convex parts.
+Supports: !arm
 
 Feature: opencl
 Description: Builds opencl enabled lib
+Build-Depends: opencl
 
 Feature: openmp
 Description: Builds openmp enabled lib

--- a/ports/v-hacd/portfile.cmake
+++ b/ports/v-hacd/portfile.cmake
@@ -1,4 +1,7 @@
+vcpkg_fail_port_install(ON_ARCH "arm")
+
 vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH

